### PR TITLE
iocage.8 updates March 2019

### PR DESCRIPTION
**Pull request by Mrt134**
Update iocage.8 according to changes from these PRs:

- #847 
- #854 
- #867 
- #868 
- #869 
- #878 
- #882 
- #884 

References these FreeNAS Redmine tickets:

- 73740
- 75360
- 75450
- 75468
- 77421
- 77574

Rearchive updated iocage.8
Syntax and flag order will need review.

Make sure to follow and check these boxes before submitting a PR! Thank you.

- [x] Explain the feature
- [x] Read [CONTRIBUTING.md](https://github.com/iocage/iocage/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: iocage.8 [hunk 1]
```diff
@@ -1015,6 +1046,30 @@ The Source listed with each property shows whether it is a local
 .Nm
 property or where more information can be located.
 .Bl -tag -width "pkglist=none"
+.It Pf assign_localhost= Op 0 | 1
+Boolean option to add interface
+.Ql lo0
+assigned to
+.Ql 127.0.0.1 .
```
Comment (skarekrow) [inline iocage.8]: It starts at 127.0.0.2 and works up tot he next available address. 127.0.0.1 is not assigned.

File: iocage.8 [hunk 2]
```diff
@@ -1015,6 +1046,30 @@ The Source listed with each property shows whether it is a local
...
+Configured automatically when
+.Ql vnet=on.
```
Comment (skarekrow) [inline iocage.8]: We should reword this as vnet jails setup their own localhost as they are an independent stack. The current text implies vnet=on sets this property. Which isn’t the case

**Review by skarekrow — CHANGES_REQUESTED** [review]
Looks good! Few changes requested, do you think we can tackle the booleans  changing in this pr as well?

1/0, yes/no, true/false, on/off are all equivalent for many of the properties now. Case does not matter

**Comment by Mrt134** [comment]
Sure, I can update the booleans too, is there a specific syntax you want that applies to all properties?

**Comment by skarekrow** [comment]
Perhaps we use 1/0 in the docs and make a note that all of those other values can also apply. Just makes things more consistent.

**Comment by Mrt134** [comment]
Ok, I should have the PR updated in just a bit.

File: iocage.8 [hunk 3]
```diff
@@ -1791,7 +1815,7 @@ Can be set to any string if needed.
 Default: the host's release
 .Pp
 Source: local
-.It Pf compression= Op on | off | lzjb | gzip | gzip-N | zle | lz4
+.It Pf compression= Op 1 | 0 Op lzjb | gzip | gzip-N | zle | lz4
```
Comment (skarekrow) [inline iocage.8]: Compression and dedup don’t apply with the Boolean changes

**Review by skarekrow — CHANGES_REQUESTED** [review]

File: iocage.8 (hunk 3 above)
Comment (Mrt134) [reply iocage.8]: whoops, thanks. I'll get that changed back.

**Comment by skarekrow** [comment]
```
            'bpf',
            'template',
            'host_time',
            'basejail',
            'dhcp',
            'vnet',
            'rtsold',
            'jail_zfs',
            'hostid_strict_check',
            'boot',
            'exec_clean',
            'mount_linprocfs',
            'mount_procfs',
            'allow_vmm',
            'allow_tun',
            'allow_socket_af',
            'allow_quotas',
            'allow_mount_zfs',
            'allow_mount_tmpfs',
            'allow_mount_procfs',
            'allow_mount_nullfs',
            'allow_mount_fusefs',
            'allow_mount_devfs',
            'allow_mount',
            'allow_mlock',
            'allow_chflags',
            'allow_raw_sockets',
            'allow_sysvipc',
            'allow_set_hostname',
            'mount_fdescfs',
            'mount_devfs',
            'ip6_saddrsel',
            'ip4_saddrsel',
            'ip_hostname',
            'assign_localhost'
```

Are the boolean props :)

**Review by skarekrow — APPROVED** [review]

**Comment by skarekrow** [comment]
Thanks Tim!
